services/demo/tsconfig.json

```diff
@@ -3,6 +3,10 @@
     "compilerOptions": {
         "baseUrl": "./",
         "outDir": "./dist",
+        "paths": {
```

Comment:
my webpack just throws compiling error without this...

Reply:
Interesting, I wish understood why this doesn't happen to me. Thanks for fixing it though